Explorer/Assets/DCL/SDKComponents/TriggerArea/Components/SDKTriggerAreaComponent.cs

```diff
@@ -0,0 +1,7 @@
+namespace DCL.SDKComponents.TriggerArea.Components
+{
+    public struct SDKTriggerAreaComponent
```

Comment:
You could delete such components-satellites. 
- On adding you can always check by the presence of the common `SDKEntityTriggerAreaComponent` as they are instantiated at the same time
- `World.Remove<SDKTriggerAreaComponent>(entity)` where it's currently doesn't lead to anything at all

I see the sense of having `AvatarModifierAreaComponent` as it carries additional data but having an empty marker doesn't seem to bring any value

Reply:
You are right, I missed that, must've been left from my initial experiments, thanks for the insight 👍  tackled at https://github.com/decentraland/unity-explorer/pull/5244/commits/5368e4ebc06c7606f44abdd052c4cb2dcc6c1a56